### DISCUSSION TITLE
Fooocus always generate photos with earrings

### DISCUSSION THREAD

**Discussion by grayistanbul** [lllyasviel/Fooocus#2469] [Q&A] [answered]
Why does Fooocus always generate photos with earrings, even though the prompt specifies not to wear ![2024-03-07_04-28-19_9308](https://github.com/lllyasviel/Fooocus/assets/162002014/a6b5b517-1b27-4f43-880e-3a58188ae9bc) or any jewelry? How can this be resolved?
![2024-03-07_04-30-44_9377](https://github.com/lllyasviel/Fooocus/assets/162002014/7885d5eb-3f79-41bf-8403-3bb6b6dc1c91)
![2024-03-07_04-53-01_1229](https://github.com/lllyasviel/Fooocus/assets/162002014/ecfef1ca-9671-4d27-96a9-ea6a43050884)

**Comment by mashb1t** [comment]
What is your positive and negative prompt?
If you do not want to have some element in the generated picture, please use the negative prompt and best use precise keywords, not natural language.

**Reply by grayistanbul** [reply]
thank you

**Comment by poor7** [comment, marked as answer]
You are probably using the Fooocus Masterpiece style which includes earrings. Disable this style.

**Reply by grayistanbul** [reply]
thank you

**Comment by wisdombox44** [comment]
it is in there fooocus masterpiece style you may disable it
use negative promt
<img width="752" alt="Screenshot 2024-03-07 215042" src="https://github.com/lllyasviel/Fooocus/assets/123548315/1d33e91e-210b-4438-939d-1f16c9755a84">
or alternatively youcan edit this style with note pad. make sure you make a backup
"C:\Your disk\Fooocus\Fooocus\sdxl_styles"

**Reply by grayistanbul** [reply]
thank you
